tests/common/unittest_common.cpp

```diff
@@ -445,7 +445,8 @@ static gchar *create_null_file (const gchar *dir, const gchar *file)
   gchar *fullpath = g_build_path ("/", dir, file, NULL);
   FILE *fp = g_fopen (fullpath, "w");
 
-  fclose (fp);
+  if(fp)
+    fclose (fp);
```

Comment:
Let's return some other string when fp is NULL.
e.g.,
```
else {
  g_free (fullpath);
  fullpath = g_strdup ("Cannot create the file.");
}
```

Reply:
> `else {
>   g_free (fullpath);
>   fullpath = g_strdup ("Cannot create the file.");
> }`

Returning this can be confusing as caller might usually just check for NULL.
I think returning NULL is better option.

Reply:
done.